examples/introductory/clipping_planes.py

```diff
@@ -38,8 +38,14 @@ def create_scene(clipping_planes, clipping_mode):
 scene1 = create_scene(clipping_planes, "any")
 scene2 = create_scene(clipping_planes, "all")
 
-scene1.add(gfx.AmbientLight(), gfx.DirectionalLight(position=(1, 2, 3)))
-scene2.add(gfx.AmbientLight(), gfx.DirectionalLight(position=(1, 2, 3)))
+light = gfx.DirectionalLight()
+light.local.position = (1, 2, 3)
+scene1.add(gfx.AmbientLight(), light)
+
+# can we reuse the light from above?
```

Comment:
I guess this depends; if `scene1` kept a reference to `light` by weakref, it would be garbage collected... probably best not to shadow the variable like this

Reply:
We can't actually, my bad. WorldObjects can only have up to 1 parent. This would assign the object to multiple parents, which isn't supported by the current transform mechanism.